### PR TITLE
initial support alpine linux

### PR DESCRIPTION
**Pull request by General-Beck**
x86_64-alpine-linux-musl

### CODE REVIEW

**Comment by gavofyork** [comment]
doesn't build.

**Comment by misery** [comment]
Any progress here? I like to add a parity package to alpine. After 1.7.0 the libressl issue is fixed but it has another problem.

See: https://github.com/alpinelinux/aports/pull/1643

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -409,6 +409,20 @@ linux-aarch64:
     - target/aarch64-unknown-linux-gnu/release/parity
     name: "aarch64-unknown-linux-gnu_parity"
   allow_failure: true
+linux-alpine:
+  stage: build
+  image: parity/rust-alpine:gitlab-ci
+  before_script:
+    - git submodule update --init --recursive
+  script:
+    - cargo build -j 4 --features final --release
```
Comment (5chdn) [inline .gitlab-ci.yml]: does alpine have `$(nproc)` ?

**Comment by arkpar** [comment]
see https://github.com/paritytech/parity/issues/5794

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by ddorgan** [comment]
[clabot:check]

**Comment by debris** [comment]
This pr hasn't been updated for > 28 days and I'm closing it as a part of pull request cleanup. If it's still relevant, please rebase and reopen it.

**Pull request closed without merging**